cypress/e2e/content/content.spec.js

```diff
@@ -145,19 +145,23 @@ describe("Content Specs", () => {
     });
 
     it("Dropdown Field", () => {
-      cy.get("#12-f3152c-kjz88l .Select")
-        .click()
+      cy.get("#12-f3152c-kjz88l")
+        .should("exist")
```

Comment:
The usage of `.should("exist")` statements after a `get()` feels superfluous to me. The `get` selector implicitly creates a should exist expectation due to the fact if get selection fails then it is not in the DOM and the test fails, as would be expected. Thoughts?

Reply:
Yea 100% agree not sure how this got added here as it does nothing.